resources/public/api/conf/1.0/common/errors.yaml

```
@@ -147,6 +147,30 @@ components:
         code: RULE_MISSING_TRADING_NAME
         message: tradingName not provided for the self-employment business
 
+    formatStatus:
+      description: The format of the supplied status is not valid.
+      value:
+        code: FORMAT_STATUS
+        message: The provided status is invalid
+
+    formatStatusReason:
+      description: The format of the supplied status reason is not valid.
+      value:
+        code: FORMAT_STATUS_REASON
+        message: The provided status reason is invalid
+
+    formatBusinessIncomePriorTo2Years:
+      description: The format of the supplied businessIncomePriorTo2Years is not valid.
+      value:
+        code: FORMAT_BUSINESS_INCOME_PRIOR_TO_2_YEARS
+        message: The provided business Income Prior  To 2 Years is invalid
```

Noticed an extra space
```suggestion
        message: The provided business Income Prior To 2 Years is invalid
```